FBC/DebtAndEquities/Debt.rdf

```diff
@@ -628,6 +636,12 @@
 		<fibo-fnd-utl-av:explanatoryNote>Note that in most cases, the dates and payment frequencies for interest will coincide with the dates and payment frequencies related to the principal.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fbc-dae-dbt;InterestRateCap">
```

Comment:
This does not seem right as a class: surely the only class needed is InterestRate and whether it's treated as a cap or floor should be determined by how it's used in properties e.g. hasInterestRateCap.

Reply:
@rivettp The reason that I created a class for this is that financial institutions are required to report on this specific item as a component of their regulatory reports monthly to a myriad of regulators.  It also factors into all kinds of analyses related to cash flows / earnings over time.  Otherwise I would have modeled it as you suggest.  Based on my experience at WF, it is far easier to map concepts like this to various repos if there is a class to map to rather than limiting the model to use of a property.